### ISSUE TITLE
DEPRECATED!

### ISSUE THREAD

**Issue by valendesigns** [envato/envato-wordpress-toolkit#85]
Hi Everyone,

Sorry for not responding to these issues, but I've been working on something new. This plugin is no longer being developed, and the old Envato API it uses is being deprecated and will no longer work at all soon. However, I'm currently working on a brand new code base that will replace this plugin, and the library, which will use the new Envato API with OAuth. It will be available around the end of the month and have PHPUnit testing, Grunt, & other build tools that will make it a much better plugin. If you're interested in beta testing, or contributing in any way, please send me your info.

**Comment by stuttboksa** [comment]
So I just have to wait without a working mobile theme...?

**Comment by valendesigns** [comment]
What do you mean, how is your theme not working?

**Comment by stuttboksa** [comment]
The below text is a copy of my own question earlier on this page:
"Hi!
I have installed the plugin and activated it. When I try to open it the page loads really slow and after a while i get the following message: 
"Error 503 Service Unavailable
Service Unavailable
XID: 1048685307"
It is 100% related to the plugin because it only happens when it's installed.
I use a theme called Native Church.
elimtistedal.no"

**Comment by valendesigns** [comment]
The plugin is not working right now because of API issues. Do not use the plugin. Problem solved.

**Comment by stuttboksa** [comment]
How can I update to v2.4.5 without the plugin?

**Comment by valendesigns** [comment]
Download the theme and install it over the current one.

**Comment by stuttboksa** [comment]
I think the one thats available for normal download is the 2.4.4...

**Comment by valendesigns** [comment]
That's something you should contact the theme author about. In the meantime, deactivate the plugin.

**Comment by stuttboksa** [comment]
i thought so but I'm a bit inpatient waiting for answers from them! Thanks!

**Comment by valendesigns** [comment]
Developers are working on a server side fix to get the API connecting with the Toolkit again. I'll know more in probably 12 hours. Cheers!

**Comment by stuttboksa** [comment]
Perfect! Where will you post the new knowledge?

**Comment by valendesigns** [comment]
Here.

**Comment by janvonde** [comment]
@valendesigns Thumbs up! 8-) I'm getting a beer and lay back...

**Comment by PosseNinja** [comment]
Fun stuff... The deprecation is affecting different sites in different manners on our server. Sites all function on the front end, but can't access the Dashboard on many of them. When we try, we get a 404. Some of those that can access it are getting an Recv failure: Connection reset by peer error. Fun stuff lol. Going through via ftp and disabling on all the sites we host now... @janvonde - 49 sites later and I might need a beer too!

**Comment by valendesigns** [comment]
Unfortunately, I wasn't aware the API was going to change yet. It's suppose to be available still, but I guess changes were made, so they're looking into what can be done server side without forcing a plugin upgrade.  I'm heading to sleep I'll let you guys know what my inbox knows when I get up. Cheers!

**Comment by PosseNinja** [comment]
no worries on our end... it's a simple fix via FTP to just turn the plugin off. Get some rest & enjoy a bit of relative peace! :)

**Comment by TimeAssistant** [comment]
Thanks for not notifying us that your plugin is now useless - 4 hours I have been going through our company website trying to find whats wrong with it as it takes 5min+ to load any wp-admin page, I googled the " Recv failure: Connection reset by peer " problem and nothing comes up that was useful. Only after I disabled every single plugin and reactivated 1 by 1 did I notice that it was Envato toolkit that broke the site.

Very poor indeed.

**Comment by PosseNinja** [comment]
@TimeAssistant the very first thing to do in troubleshooting WP is to disable all plugins. It literally took 5 minutes to figure out what the issue was on our sites. 4 hours before trying to disable plugins is a long time... just sayin.

**Comment by TimeAssistant** [comment]
@PosseNinja  Yeah, thats great when you can actually access your wp-admin to figure it out - when the site keeps timing out or takes 5min+ to load pages its hard to find out whats up. Not only that disabling all our plugins breaks the front end of the site as it would with most, and as its the company website a customer facing site, its important to keep the thing working (especially as the front end was working at the time). I also have 20+ plugins on the site so it also took ages to disable these and I prefer going through 1 by 1 as manually disabling them all via ftp can require that some plugins are configured again or reconnected to external services. 4 Hours included disabling plugins, but it also includes time researching. Any time a website has a fault if I get an error message especially in admin panel the first thing I would do is to search the problem. Not all wordpress issues are plugin related....  I dont just decide to destroy our websites by deactivating every plugin when something goes wrong ..... "just sayin"

**Comment by dansully** [comment]
This has been a really helpful plugin for over a year! I keep it turned off and only turn it on when I get the email notice that an update is available. When I want to run the update I turn on the plugin, run the update, and turn it back off. Thanks for working on this. I'm just trying to outweigh the people that are mad that a FREE plugin made by a VOLUNTEER isn't working right now. :D Keep up the good work!

**Comment by TimeAssistant** [comment]
@dansully maybe people are mad that a FREE plugin made by a volunteer BROKE and DISABLED the wp-admin pages on their site which for me caused further breakages of the site in regards to having to disable every plugin to find out whats wron.. Not to mention I never volunteered to install this, our purchases from Envato kept saying it was a required plugin and kept prompting for install, currently trying to get me to reinstall it now aswell... it may have been a good plugin but users should have been urgently notified of the fact its depreciated now.

**Comment by dansully** [comment]
@TimeAssistant It *is* frustrating, but don't take it out on the guy making the plugin.

**Comment by PosseNinja** [comment]
@manually you get a big +1 in my book. :) The plugin author didn't force Envato to make it a required plugin. And you nailed it. Dude created a FREE plugin that is a check of a convenience. I manage 49 sites and have definitely appreciated this plugin.

**Comment by AndonDesign** [comment]
Hi guys, yesterday I posted a potential work around for the API connectivity issue, it fixed it for my theme anyways, might be worth to try:
https://github.com/envato/envato-wordpress-toolkit/issues/82#issuecomment-118649827

Cheers,
Andon

**Comment by TimeAssistant** [comment]
@dansully  I'm not necessarily taking it out on him, but what I am saying is that as the author its their responsibility to let users know if the plugin has came to the end of its life, or if its going to create massive problems etc. For one, he could have had it trigger an email to site admins, or email envato accounts anything to let us know prior to finding out the hard way. 

1 small email with some information could have saved me around 4 hours and other countless minutes / hours that is my point. In a job where time is money this is a bit of a problem


@PosseNinja  It doesnt matter how many sites someone manages, I manage many myself and create/build them. So do you expect all free plugins to break your back end functionality? I certainly don't which is the problem here, yes it was a helpful plugin and yes it streamlined the process of updating themes etc from envato. However breaking your admin backend and leaving you locked out your site is very bad, especially when all the time could have been saved by a small simple email message. If I was the author of a plugin I would make sure I knew if the API I had used was about to be took off.

What I'm getting at is his post here was useless the only people to read it will be the ones who have already figured out its envato toolkit causing issues and it needs disabled.

**Comment by PosseNinja** [comment]
Thanks for the workaround @And on :)

**Comment by valendesigns** [comment]
Hey Everyone,

I still don't have any new information to share, but as soon as I do I will post it here. I'm going to start reaching out to the API devs but it's still very early in Australia so they are not in the office yet.

@TimeAssistant What do you think I'm doing here? I'm notifying you that the plugin is not working and is no longer being developed. I don't control the Envato API and what they do with it. How am I suppose to give you a heads up about something I don't know. Seems unreasonable to me, I'm not psychic, nor am I responsible for the connectivity of the Envato API. This plugin was already being deprecated before all this API nonsense happened or I was made aware there was an issue. It's just bad timing.

**Comment by valendesigns** [comment]
I've created a new pull request [hotfix-1.7.3](https://github.com/envato/envato-wordpress-toolkit/pull/86) that will be merged into master within the hour. Please test it if you can. It should solve the API connectivity issue in the interim while the new plugin is being developed.

**Comment by TimeAssistant** [comment]
@valendesigns my point is only the people checking here will find the answer... Not everyone will know to check github.

**Comment by dynalabs** [comment]
Hi Guys,

Thank you for the plugin. I've installed it on 2 sites. One was about a week ago and it still seems to be working fine still.

The second was yesterday and it does not recognise that my theme is installed.

Is it safe to assume that this is because the plugin is deprecated and I should deactivate it on both sites?

Thanks again

**Comment by AGSoftAU** [comment]
Any news @valendesigns?

**Comment by valendesigns** [comment]
I would love to have good news. However, the new API does not have the methods required for the plugin to work. I'm waiting for the Envato Market development team to do their part. As soon as they add the methods that were in the previous API work can resume. Sorry guys!

**Comment by leowys** [comment]
Hello Derek! Is there any news about this new plugin you are developing to fix / replace Envato Toolkit? Thanks!

**Comment by valendesigns** [comment]
Here is the replacement plugin, I should have posted this sooner.

https://github.com/envato/wp-envato-market

**Comment by leowys** [comment]
Thanks! Great work!
It's working great in the 2 or 3 sites i've tested.
Please keep it alive with further updates.

**Comment by sswdo** [comment]
I removed the old plugin some time ago as I was getting an API error message and installed the new envato market plugin. Yet I am still getting an Envato APi error messages when I update any plugins. It says the plugin has failed to update but if you refresh the page the plugin would have been updated. I deactivated the theme and 2 plugins from envato but the error is still showing up. I went into MYSQL db but I cannot figure out what are the tables (I thought maybe by deleting them it would get rid of the error). Attached is a screenshot of the type of errors I am getting. How do I get rid of it?
<img width="1592" alt="screen shot 2016-11-10 at 7 32 45 am" src="https://cloud.githubusercontent.com/assets/24687938/21374220/e89a730c-c770-11e6-9d1c-53128c6065ea.png">

**Comment by cyberlee1** [comment]
hello everyone
